projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -481,7 +481,25 @@ export class RootComponent implements OnInit, OnDestroy {
           downloadAnchorNode.click();
           downloadAnchorNode.remove();
         });
-    } else {
+    } else if (option === 'OWL') {
+
+      this.sheetService
+        .fetchSheetData(sheet.sheetId, sheet.gid, csvURL? csvURL : sheet.csvUrl, null, 'owl')
+        .subscribe((graphData: any) => {
+          const graphDataStr =
+            'data:text/json;charset=utf-8,' +
+            encodeURIComponent(JSON.stringify(graphData));
```

Comment:
You shouldn't need to stringify the data. Also the response is string (not json) so this is currently failing.